tracing-journald/src/lib.rs

```diff
@@ -277,16 +288,19 @@ struct SpanFields(Vec<u8>);
 struct SpanVisitor<'a> {
     buf: &'a mut Vec<u8>,
     depth: usize,
-    prefix: Option<&'a str>,
+    field_prefix: Option<&'a str>,
+    span_field_prefix: Option<&'a str>,
 }
 
 impl SpanVisitor<'_> {
     fn put_span_prefix(&mut self) {
-        write!(self.buf, "S{}", self.depth).unwrap();
-        if let Some(prefix) = self.prefix {
-            self.buf.extend_from_slice(prefix.as_bytes());
+        if let Some(span_prefix) = self.span_field_prefix {
+            write!(self.buf, "{}{}", span_prefix, self.depth).unwrap();
+            if let Some(prefix) = self.field_prefix {
+                self.buf.extend_from_slice(prefix.as_bytes());
+            }
```

Comment:
it looks like if `span_field_prefix` is `None`, we will also skip emitting the field prefix. I'm not sure if that is correct --- this would mean there is no way to disable the span field prefix without also disabling the field prefix. i _think_ these should be configured separately.

Reply:
Sure, that's my oversight.